views/bigquery.go

```diff
@@ -155,18 +155,18 @@ func (bq *bigqueryHandler) QueryViewsEvents(ctx context.Context, spec QuerySpec)
 func buildViewsEventsQuery(table string, spec QuerySpec) (string, []interface{}, error) {
 	query := squirrel.Select(
 		"countif(play_intent) as view_count",
```

Comment:
Shouldn't this have the if_null as well? Thought it was having issues too

Reply:
No need, `countif()` counts all True expressions, NULL is not True, so, it'll be counted as `0`.